### PR TITLE
Student:FeedbackSubmissionEditPage incomplete display of question title #8505

### PR DESCRIPTION
**Pull request by Progyan1997**
Wrap question text around span element.

Fixes: #8505
Closes: #8610

**PR Checklist**

Ensure that you have:
- [ ] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [ ] Added the issue number to the "Fixes" keyword above
  - [ ] Titled the PR as specified in the abovementioned document
- [ ] Made your changes on a branch other than `master` and `release`
- [ ] Gone through all the changes in this PR and ensured that:
  - [ ] They addressed one (and only one) issue
  - [ ] No unintended changes were made
- [ ] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [ ] Added/updated tests, if changes in functionality were involved
- [ ] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

### CODE REVIEW

**Comment by Progyan1997** [comment]
@bqnguyen94 I found these following failed case in test 4:

70c70
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
144c144
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
219c219
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
378c378
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
495c495
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
612c612
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
686c686
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">

	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at teammates.test.driver.HtmlHelper.assertSameHtml(HtmlHelper.java:95)
	at teammates.test.driver.HtmlHelper.assertSameHtml(HtmlHelper.java:63)
	at teammates.test.pageobjects.AppPage.verifyHtmlPart(AppPage.java:870)
	at teammates.test.pageobjects.AppPage.verifyHtmlMainContent(AppPage.java:922)
	at teammates.test.cases.browsertests.FeedbackRankQuestionUiTest.testStudentSubmitAndResultsPages(FeedbackRankQuestionUiTest.java:208)
	...

> Failed suite [Failed suite [Failed suite [ci-tests]]] > parallel-ui-tests(failed)(failed)(failed) > teammates.test.cases.browsertests.FeedbackRankQuestionUiTest.testStudentSubmitAndResultsPages FAILED
    java.lang.AssertionError at FeedbackRankQuestionUiTest.java:208

70c70
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
218c218
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
381c381
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
473c473
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
511c511
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">

	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at teammates.test.driver.HtmlHelper.assertSameHtml(HtmlHelper.java:95)
	at teammates.test.driver.HtmlHelper.assertSameHtml(HtmlHelper.java:63)
	at teammates.test.pageobjects.AppPage.verifyHtmlPart(AppPage.java:870)
	at teammates.test.pageobjects.AppPage.verifyHtmlMainContent(AppPage.java:922)
	at teammates.test.cases.browsertests.AdminHomePageUiTest.testCreateInstructorAction(AdminHomePageUiTest.java:235)
	at teammates.test.cases.browsertests.AdminHomePageUiTest.testAll(AdminHomePageUiTest.java:53)
	...

> Failed suite [Failed suite [Failed suite [ci-tests]]] > sequential-ui-tests(failed)(failed)(failed) > teammates.test.cases.browsertests.AdminHomePageUiTest.testAll FAILED
    java.lang.AssertionError at AdminHomePageUiTest.java:235

> org.gradle.internal.serialize.PlaceholderException: Unable to locate element: {"method":"class name","selector":"bootbox-close-button"}
Command duration or timeout: 56 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'travis-job-e511c681-48c4-474b-8aa6-454d6a5ad322', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.0-101-generic', java.version: '1.8.0_151'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=46.0, platform=LINUX, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: d7a6ff8b-88cc-44d0-ba80-de9005d5934a
*** Element info: {Using=class name, value=bootbox-close-button}
	at sun.reflect.GeneratedConstructorAccessor25.newInstance(null:-1)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByClassName(RemoteWebDriver.java:477)
	at org.openqa.selenium.By$ByClassName.findElement(By.java:391)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at teammates.test.pageobjects.AppPage.waitForModalPresence(AppPage.java:322)
	at teammates.test.pageobjects.AppPage.clickAndConfirmWithoutWaitingForModalDisappearance(AppPage.java:684)
	at teammates.test.pageobjects.InstructorCourseStudentDetailsEditPage.fillStudentDetailsForm(InstructorCourseStudentDetailsEditPage.java:84)
	at teammates.test.pageobjects.InstructorCourseStudentDetailsEditPage.submitUnsuccessfully(InstructorCourseStudentDetailsEditPage.java:54)
	at teammates.test.cases.browsertests.InstructorCourseStudentDetailsEditPageUiTest.testInputValidation(InstructorCourseStudentDetailsEditPageUiTest.java:73)
	at teammates.test.cases.browsertests.InstructorCourseStudentDetailsEditPageUiTest.testAll(InstructorCourseStudentDetailsEditPageUiTest.java:29)
	...

> Failed suite [Failed suite [Failed suite [ci-tests]]] > parallel-ui-tests(failed)(failed)(failed) > teammates.test.cases.browsertests.InstructorCourseStudentDetailsEditPageUiTest.testAll FAILED
    org.openqa.selenium.NoSuchElementException at InstructorCourseStudentDetailsEditPageUiTest.java:73
        Caused by: org.openqa.selenium.NoSuchElementException

81c81
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
153c153
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">
326c326
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">

	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at teammates.test.driver.HtmlHelper.assertSameHtml(HtmlHelper.java:95)
	at teammates.test.driver.HtmlHelper.assertSameHtml(HtmlHelper.java:63)
	at teammates.test.pageobjects.AppPage.verifyHtmlPart(AppPage.java:870)
	at teammates.test.pageobjects.AppPage.verifyHtmlMainContent(AppPage.java:922)
	at teammates.test.cases.browsertests.StudentFeedbackResultsPageUiTest.testAll(StudentFeedbackResultsPageUiTest.java:38)
	...

> Failed suite [Failed suite [Failed suite [ci-tests]]] > parallel-ui-tests(failed)(failed)(failed) > teammates.test.cases.browsertests.StudentFeedbackResultsPageUiTest.testAll FAILED
    java.lang.AssertionError at StudentFeedbackResultsPageUiTest.java:38

70c70
<         <span class="text-preserve-space">
---
>         <span class="text-preserve-space word-wrap-break">

	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at teammates.test.driver.HtmlHelper.assertSameHtml(HtmlHelper.java:95)
	at teammates.test.driver.HtmlHelper.assertSameHtml(HtmlHelper.java:63)
	at teammates.test.pageobjects.AppPage.verifyHtmlPart(AppPage.java:870)
	at teammates.test.pageobjects.AppPage.verifyHtmlMainContent(AppPage.java:922)
	at teammates.test.cases.browsertests.FeedbackRubricQuestionUiTest.testStudentResultsPage(FeedbackRubricQuestionUiTest.java:55)
	at teammates.test.cases.browsertests.FeedbackRubricQuestionUiTest.allTests(FeedbackRubricQuestionUiTest.java:46)
	...

> Failed suite [Failed suite [Failed suite [ci-tests]]] > parallel-ui-tests(failed)(failed)(failed) > teammates.test.cases.browsertests.FeedbackRubricQuestionUiTest.allTests FAILED
    java.lang.AssertionError at FeedbackRubricQuestionUiTest.java:55

File: src/test/resources/test.template.properties [hunk 1]
```diff
@@ -43,7 +43,7 @@ test.timeout=15
 # Set to true to enable GodMode.
 # Please read the GodMode documentation if you are not yet familiar with it, and use with care.
 # Remember to set back to false when done and rerun the test(s).
-test.godmode.enabled=false
```
Comment (bqnguyen94) [inline src/test/resources/test.template.properties]: You don't push this file. To turn on `God Mode`, change `test.properties` instead.

**Comment by bqnguyen94** [comment]
Those conflicts are because the expected HTML files don't contain `word-wrap-break`. You need to update them either manually one by one, or run the relevant tests with `God Mode` on.

File: src/test/resources/test.template.properties (hunk 1 above)
Comment (Progyan1997) [reply src/test/resources/test.template.properties]: @bqnguyen94 I was thinking of doing the changes and rebase before asking for review.

**Comment by damithc** [comment]
> You need to update them either manually one by one, or run the relevant tests with God Mode on.

In fact manual modification is not recommended. Use the God mode instead. :-)

**Comment by Progyan1997** [comment]
@bqnguyen94 I am still not getting there. Can you guide me through? :cry:

**Comment by wkurniawan07** [comment]
@Progyan1997 what exactly have you tried (and possibly failed)? Based on the files changed it does not look like you have fixed anything with God mode.

**Comment by Progyan1997** [comment]
@wkurniawan07 I tried enabling GodMode in `test.properties` but it updated some other pages instead of what I changed.

**Comment by wkurniawan07** [comment]
@Progyan1997 you can try to push it here. At least we can have a look.

**Comment by wkurniawan07** [comment]
@Progyan1997 anything new here?

**Pull request closed without merging**